TelecomColorCodeUtility.h

```diff
@@ -0,0 +1,28 @@
+#include "TelecomColorCode.h"
```

Comment:
Filenames like `Utility` keep attracting unrelated functionality. Here for example, the reporting functionality is likely to evolve, with customers asking for different formats, etc. Perhaps there would be need to publish on the web as well. Everything will land in the utility file. Try naming it more specific to express your intent

Reply:
updated